src/test/app/Regression_test.cpp

```diff
@@ -165,10 +165,11 @@ struct Regression_test : public beast::unit_test::suite
     {
         testcase("Autofilled fee should use the escalated fee");
         using namespace jtx;
-        Env env(*this, envconfig([](Config& cfg)
+        Env env(*this, envconfig([](std::unique_ptr<Config> cfg)
```

Comment:
We could make the params to these lambdas `auto`, for terseness. No change needed.